app/src/main/java/ar/com/wolox/android/example/model/User.java

```diff
@@ -0,0 +1,21 @@
+package ar.com.wolox.android.example.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ *
+ */
+public class User {
+    @SerializedName("email")
```

Comment:
Please add a blanck line above, and write attributes in one line like this:
@SerializedName("email) private String email;